### ISSUE TITLE
Mining tab not showing corectly on versions above v5.0.0

### ISSUE THREAD

**Issue by Waytoogo** [btdex/btdex#176]
I'm running Windows 10 21H1. I have the 64 bit Java 8.0.291 installed. I have tried both the exe file and the jar file with the same results. In version 5.0.0 my mining tab looks like this. It shows everything including my projected earnings.
![v0 5 0](https://user-images.githubusercontent.com/73441019/124395239-f95f1b00-dcd0-11eb-8134-e5f77615ab26.JPG)
In versions 0.5.2, 0.5.3, and 0.5.4 it looks like this. On the left side I can only see 3 drives, and on the right the projected earnings are not visible. If I maximize the windows only the mining log gets taller. I hope this is something that can be easily fixed.
![v0 5 4](https://user-images.githubusercontent.com/73441019/124395240-f95f1b00-dcd0-11eb-871e-2c6616617ba2.JPG)

**Comment by jjos2372** [comment]
Hello, thanks for reporting. The projected earnings need to take into account the minimum mining size, which is currently 1 TiB, and also that the commitment only activates after 60 blocks. So it was suppressed in the latest releases until the better way to show "pending" commitment is elaborated. Otherwise, we keep getting the same questions.

**Comment by Waytoogo** [comment]
> 
> 
> Hello, thanks for reporting. The projected earnings need to take into account the minimum mining size, which is currently 1 TiB, 
Does this mean that if I try to mine with only 900 GiB, It won't mine, or I won't get paid for it?

**Comment by jjos2372** [comment]
> Does this mean that if I try to mine with only 900 GiB, It won't mine, or I won't get paid for it?

No, you can mine with any capacity, it is just that in the commitment calculation the chain will assume you have 1 TiB minimum when calculating your **SINGA per TiB** commitment.
